crates/fpe/src/wasm_bindgen/float.rs

```diff
@@ -0,0 +1,43 @@
+use wasm_bindgen::{prelude::wasm_bindgen, JsValue};
+
+use crate::core::{Float, KEY_LENGTH};
+
+fn fpe(input: f64, key: Vec<u8>, tweak: Vec<u8>, encrypt_flag: bool) -> Result<f64, JsValue> {
+    if key.len() != KEY_LENGTH {
+        return Err(JsValue::from_str(&format!(
+            "FPE Float error: key length incorrect: {}, expected {}",
+            key.len(),
+            KEY_LENGTH
+        )));
+    }
+    let mut k: [u8; 32] = [0; 32];
+    k.copy_from_slice(&key);
```

Comment:
Use `TryFrom` implementation

Reply:
You can remove the check above with `try_into()`

Reply:
indeed